ui/dev/mime.types

```diff
@@ -0,0 +1,13 @@
+types {
+    text/html                             html htm shtml;
+    text/css                              css;
+    text/xml                              xml;
+    image/gif                             gif;
+    image/jpeg                            jpeg jpg;
+    application/x-javascript              js;
+    application/atom+xml                  atom;
+    application/rss+xml                   rss;
+    ..........................................
```

Comment:
Suggest to remove the placeholder.

Are these _all_ mime types used by the UI, or maybe there is a more extensive list just to be sure?